src/components/LiquidationPrice.js

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from "react";
+import Flex, {FlexItem} from "styled-flex-component";
+import ReactTooltip from "react-tooltip"
+import styled from "styled-components"
+import {device} from "../screenSizes"
+import {getLiquidationPrice} from "../lib/Actions"
```

Comment:
big G?

Reply:
it is a function not a class
Capital letters are used for class in JS (and in some other languages as well)